entities/Purpose/Purpose.ts

```diff
@@ -1,8 +1,8 @@
-import {Id, StaticEntity} from './common';
-import {SegmentName} from "./Segment";
-import {ActivityIndicatorName} from "./indicators/ActivityIndicator";
-import {ActivityFieldName, ProfileFieldName} from "./common/Field";
-import {ActivitySchemaName} from "./Schema";
+import {Id, StaticEntity} from '../common';
```

Comment:
rename this file to `index.ts` so imports could continue to be from `/Purpose`

Reply:
will do, thank you

Reply:
will do, thank you